core/frontend/src/components/system-information/NetworkCard.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <v-card
+    class="mx-auto"
+    min-width="400"
+  >
+    <v-list-item two-line>
+      <v-list-item-content>
+        <v-list-item-title class="text-h6">
+          {{ network.name }} {{ network.is_loopback ? '(loopback)' : '' }}
+        </v-list-item-title>
+        <v-list-item-subtitle v-if="network.description">
+          Description: {{ network.description }}
+        </v-list-item-subtitle>
+      </v-list-item-content>
+    </v-list-item>
+
+    <v-card-text class="pa-0">
+      <v-col
+        align="h-center"
+        class="pa-0"
+      >
+        <v-col
+          v-for="(ip, i) in network.ips"
+          :key="i"
+          class="pa-0 text-h6 text-center"
+        >
+          {{ ip }}
+        </v-col>
+      </v-col>
+    </v-card-text>
+
+    <v-list-item>
+      <v-list-item-icon>
+        <v-icon>
+          mdi-network
+        </v-icon>
+      </v-list-item-icon>
+      <v-list-item-subtitle>{{ network.is_up ? 'Enabled' : 'Disabled' }}</v-list-item-subtitle>
+    </v-list-item>
+
+    <v-list-item>
+      <v-list-item-icon>
+        <v-icon>
+          mdi-badge-account-horizontal
+        </v-icon>
+      </v-list-item-icon>
+      <v-list-item-subtitle>MAC:</v-list-item-subtitle>
+      <v-list-item-subtitle
+        class="text-right"
+        :style="show_mac ? '' : 'filter:blur(0.5em)'"
```

Comment:
Suggestion: using `cursor: pointer` css here on hover (`class:hover{}`) to change the mouse cursor, so user understand this is clickable (when blured). Right now, it's a text-selection cursor.

Not necessary, but pretty.